flask_rebar/messages.py

```diff
@@ -7,35 +7,80 @@
     :copyright: Copyright 2018 PlanGrid, Inc., see AUTHORS.
     :license: MIT, see LICENSE for details.
 """
+from collections import namedtuple
+from enum import IntEnum
 
-body_validation_failed = "JSON body parameters are invalid."
 
-empty_json_body = "Fields must be in JSON body."
+class ErrorCode(IntEnum):
+    BODY_VALIDATION_FAILED = 1
```

Comment:
Thoughts on using strings for error codes?

E.g.:
```python
class ErrorCode(IntEnum):
    BODY_VALIDATION_FAILED = "body_validation_failed"
    EMPTY_JSON_BODY = "empty_json_body"
```

I imagine these are mostly intended to be used by clients that want to programmatically respond to errors (e.g. render a translated error message on a UI somewhere), so I don't think the value matters that much, as long as it's unique. Might as well make it a little more human friendly... might help clients avoid some simple coding errors.

Reply:
Great point - will adjust